DEBUG 2020-09-02 21:47:20.101 UTC: 	 CRAB Client version: v3.200812
DEBUG 2020-09-02 21:47:20.102 UTC: 	 Running on: Linux t3.unl.edu 3.10.0-1062.9.1.el7.x86_64 #1 SMP Fri Dec 6 15:49:49 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-09-02 21:47:20.102 UTC: 	 Executing command: 'submit'
DEBUG 2020-09-02 21:47:20.102 UTC: 	 Found CRAB cache file /home/t3-ku/janguian/.crab3
INFO 2020-09-02 21:47:20.116 UTC: 	 Will use CRAB configuration file jpsidata_crab.py
DEBUG 2020-09-02 21:47:20.116 UTC: 	 Loading CRAB configuration file.
ERROR 2020-09-02 21:47:20.138 UTC: 	 Error while loading CRAB configuration:
cannot import name getUsernameFromSiteDB
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2020-09-02 21:47:20.138 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/submit.py", line 35, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 311, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 88, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
cannot import name getUsernameFromSiteDB
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2020-09-02 21:47:20.144 UTC: 	 Log file is /home/t3-ku/janguian/CMSSW_10_2_5/src/MuonAnalysis/TagAndProbe/test/susy/crab/JpsiData/crab.log
DEBUG 2021-01-12 22:42:34.023 UTC: 	 CRAB Client version: v3.201110
DEBUG 2021-01-12 22:42:34.023 UTC: 	 Running on: Linux t3.unl.edu 3.10.0-1062.9.1.el7.x86_64 #1 SMP Fri Dec 6 15:49:49 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2021-01-12 22:42:34.023 UTC: 	 Executing command: 'submit'
DEBUG 2021-01-12 22:42:34.024 UTC: 	 Found CRAB cache file /home/t3-ku/janguian/.crab3
INFO 2021-01-12 22:42:34.027 UTC: 	 Will use CRAB configuration file jpsidata_crab_2016.py
DEBUG 2021-01-12 22:42:34.027 UTC: 	 Loading CRAB configuration file.
ERROR 2021-01-12 22:42:34.042 UTC: 	 Error while loading CRAB configuration:
name 'cnfig' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2021-01-12 22:42:34.043 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.201110.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.201110.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.201110.01/lib/CRABClient/Commands/submit.py", line 35, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.201110.01/lib/CRABClient/Commands/SubCommand.py", line 312, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.201110.01/lib/CRABClient/Commands/SubCommand.py", line 90, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
name 'cnfig' is not defined
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2021-01-12 22:42:34.046 UTC: 	 Log file is /home/t3-ku/janguian/CMSSW_10_2_5/src/MuonAnalysis/TagAndProbe/test/susy/crab/JpsiData/crab.log
